### PR TITLE
exclusion is not supported by notification filters

### PR DESCRIPTION
**Pull request by mchelen-gov**
### Why:

The custom filter docs do not mention this limitation.

### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging.
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by deril1 — CHANGES_REQUESTED** [review]
content/github/managing-subscriptions-and-notifications-on-github/managing-notifications-from-your-inbox.md

**Comment by janiceilene** [comment]
Thanks so much @mchelen-gov! I'll get this triaged for review 🌟

File: content/github/managing-subscriptions-and-notifications-on-github/managing-notifications-from-your-inbox.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ Custom filters do not currently support:
   - Distinguishing between the `is:issue`, `is:pr`, and `is:pull-request` query filters. These queries will return both issues and pull requests.
   - Creating more than 15 custom filters.
   - Changing the default filters or their order.
+  - Search [exclusion](https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/understanding-the-search-syntax#exclude-certain-results) using `NOT` or `-QUALIFIER`.
```
Comment (hubwriter) [inline content/github/managing-subscriptions-and-notifications-on-github/managing-notifications-from-your-inbox.md:84]: Many thanks for adding this.

To accommodate page versioning, we leave off the first part of URLs in links to other GitHub docs pages: 
```suggestion
  - Search [exclusion](/github/searching-for-information-on-github/understanding-the-search-syntax#exclude-certain-results) using `NOT` or `-QUALIFIER`.
```

**Review by hubwriter — CHANGES_REQUESTED** [review]
@mchelen-gov - Thanks for taking the time to submit this pull request. We really appreciate your help improving GitHub documentation. I've suggested a small change. Thanks again. 👍

**Comment by mchelen-gov** [comment]
commit suggestion, thanks!

**Review by hubwriter — APPROVED** [review]

**Comment by hubwriter** [comment]
Thanks @mchelen-gov - merging this now.

**Pull request merged**